CONTRIBUTING.md

```diff
@@ -73,18 +73,9 @@ To add or update a go dependency:
   make test PKG=./kv TESTS='^TestFoo' TESTTIMEOUT=10s
   ```
 
-  When you're ready to commit, do just that with a succinct title and
-  informative message. For example,
-
-  ```bash
-  $ git commit
-  > 'update CONTRIBUTING.md
-  >
-  > Added details on running specific tests via `make`, and
-  > the CircleCI-equivalent test suite.
-  >
-  > Fixed some formatting.'
-  ```
+  When you're ready to commit, be sure to write a Good Commit Message™. Consult
+  https://github.com/erlang/otp/wiki/Writing-good-commit-messages if you're
+  not sure what constitutes a Good Commit Message™.
```

Comment:
Perhaps we should add the rule about prefixing the changed package(s) to the commit message here.